pkg/preflights/host-preflight.yaml

```
@@ -253,7 +253,7 @@ spec:
     - textAnalyze:
         checkName: "'modprobe' Command"
         fileName: host-collectors/run-host/check-modprobe.txt
-        regex: '/usr/sbin/modprobe'
+        regex: 'modprobe'
```

I would _prefer_ to have this be  `/modprobe$` instead

```suggestion
        regex: '/modprobe$'
```